backend/tracim_backend/lib/webdav/resources.py

```diff
@@ -210,7 +213,14 @@ def getMemberList(self):
 
         members = []
         for workspace in self.workspace_api.get_all():
-            workspace_path = '%s%s%s' % (self.path, '' if self.path == '/' else '/', workspace.label)
+            workspace_label = webdav_convert_file_name_to_display(workspace.label)
+
+            if not self.path.endswith('/'):
+                path = '{}/'.format(self.path)
+            else:
+                path = self.path
```

Comment:
I suggest `path = (self.path + '/').replace('//', '/')  # add trailing slash`. or even better : `path = add_trailing_slash(self.path)` with code moved to a utility function

Reply:
done.